core/core-funcs.el

```diff
@@ -140,14 +140,29 @@ Supported properties:
        `((dolist (val ',def-key)
           (define-key (eval (car val)) (kbd (cdr val)) ',func)))))))
 
+(defun spacemacs/prettify-org-buffer ()
```

Comment:
@nwolfe  Good idea. 
I'll work on this part anyway. Since hiding emphasis symbols brakes table alignment.